CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Use prometheus rules only on MCs.
```

Comment:
It looks like this is already the case as these templates are all wrapped with `{{- if .Values.global.isManagementCluster }}`. So the actual problem is the missing `cluster_type` label, right? If so, let's fix the change description.